examples/simple/example.js

```diff
@@ -5,7 +5,7 @@ const fs = require('fs')
 const path = require('path')
 const assert = require('assert')
 
-fastify.register(require('./'), {
+fastify.register(require('@fastify/secure-session'), {
```

Comment:
Please revert this (should be `../..` now). The examples are used to verify the correctness of the module while developing it.

Reply:
```suggestion
fastify.register(require('../..'), {
```